admin/pre_reg/utils.py

```diff
@@ -5,10 +5,14 @@
 
 
 def get_prereg_reviewers():
-    return MyUser.objects.filter(
-        groups__name='prereg_group'
-    ).annotate(
-        fuller_name=Concat('first_name', Value(' '), 'last_name')
-    ).values_list(
-        'email', 'fuller_name'
-    )
+    # Note - fixes django.db.utils.OperationalError: no such table' error if ever one erases a db table and tries to remigrate
```

Comment:
Gosh, why would we need to plan for that? Seems like if you're dropping tables then it's up to you to deal with it on your own. Is there a use case I'm missing here?